strongbox-data-service/src/main/java/org/carlspring/strongbox/data/service/StrongboxUserService.java

```diff
@@ -0,0 +1,13 @@
+package org.carlspring.strongbox.data.service;
+
+import org.carlspring.strongbox.data.domain.StrongboxUser;
+
+import java.util.Optional;
+
+/**
+ * @author Alex Oreshkevich
+ */
+public interface StrongboxUserService extends CrudService<StrongboxUser, String> {
```

Comment:
I think this class could just be called `UserService` (and the same goes for the `*Impl`).